yt_idefix/fields.py

```diff
@@ -20,11 +23,23 @@ def setup_fluid_fields(self):
         setup_magnetic_field_aliases(
             self, "idefix-vtk", [f"BX{idir}" for idir in "123"]
         )
+        for idir in "123":
+            self.alias(
+                ("gas", f"BX{idir}"),
+                ("idefix-vtk", f"BX{idir}"),
+                units="code_magnetic",
+            )
 
 
 class PlutoVtkFields(BaseVtkFields):
     def setup_fluid_fields(self):
         setup_magnetic_field_aliases(self, "pluto-vtk", [f"BX{idir}" for idir in "123"])
+        for idir in "123":
+            self.alias(
+                ("gas", f"BX{idir}"),
+                ("pluto-vtk", f"BX{idir}"),
+                units="code_magnetic",
+            )
```

Comment:
Following my previous comment
```suggestion
```

Reply:
Okay understood.
@neutrinoceros 
One question:
Do we keep `("pluto-vtk", "BX1")` and `("pluto-xdmf", "Bx1")` as seperate or have alias for both as `("PLUTO", "BX1")` say? I think it will be convenient for a user to use the same name for the same field across two different file formats. Let me know your opinion and I'll make changes accordingly.

Reply:
moving to a unified name (say `"pluto"`) would technically be a breaking change, so I'm not immediately sold on this.
Additionally, yt supports field type inferring when only the field name is provided (say, `"BX1"`), *on the condition that it matches one and only one field type*, so I encourage using implicit field types for on-disk field names, and explicit field-type, field-name tuples for aliases.